File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Open and save Word document in ASP.NET MVC | Syncfusion
+description: Open and save Word document in ASP.NET MVC application using Syncfusion .NET Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Open and Save Word document in ASP.NET MVC
+
+Syncfusion Essential DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, and edit **Word** documents programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **open and save a Word document in ASP.NET MVC**.
```

Comment:
Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to **create, read, and edit Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **open and save a Word document in ASP.NET MVC**.